tests/main/uc20-snap-recovery-encrypt/task.yaml

```diff
@@ -38,7 +45,7 @@ prepare: |
 execute: |
     LOOP="$(cat loop.txt)"
     echo "Run the snap-bootstrap tool"
-    /usr/lib/snapd/snap-bootstrap create-partitions --encrypt --key-file keyfile ./gadget-dir "$LOOP"
+    /usr/lib/snapd/snap-bootstrap create-partitions --encrypt  --mount --key-file keyfile ./gadget-dir "$LOOP"
```

Comment:
Since we're mounting here, we could use the mounted filesystems to test content deploy instead of double mounting.

Reply:
Indeed, nice one